### PR TITLE
Update abtesting docs

### PR DESCRIPTION
**Pull request by jrcrespoh**
- [x] Update README.md
- [x] Add OUTLINE.md

### CODE REVIEW

File: abtesting/README.md [hunk 1]
```diff
@@ -1,35 +1,65 @@
...
-to the A/B test device configuration on the Firebase Console, tap the Refresh
-button to update the UI, or alternatively on iOS 15 pull down on the screen until
-the refresh icon starts rotating.
+In the Firebase Console's A/B Testing section, click on your experiment. Under 
+'Experiment overview', click on the details (vertical dots / ellipsis) to manage your experiment, 
+then press 'Manage test devices'. Run the sample and copy the printed installation auth token from 
+Xcode's console into the text field, select 'Variant A' from the 'Variant' drop-down, and click the
+ 'Add' button. After making changes to the A/B test device configuration on the Firebase Console, 
+ tap the Refresh button to update the UI, or alternatively on iOS 15 pull down on the screen until 
+ the refresh icon starts rotating.
```
Comment (ncooke3) [inline abtesting/README.md:40]: it looks like some additional whitespace got added at the beginning of these lines

File: abtesting/README.md [hunk 2]
```diff
@@ -1,35 +1,65 @@
...
-Under 'Variant A', provide a value of `dark` to `color_scheme`. Finally, press the
-`Review` button.
+In Firebase Console's A/B Testing section, click the `Create experiment` button and select the 
+`Remote Config` option. For Basics, provide a name for the experiment and optionally a description.
+ For Targeting, select your app using the drop-down and increase the 'Exposure' to 100%. For Goals,
+  feel free to choose any metric from the drop-down, such as `Crash-free users`. For Variants, 
+  click `Choose or create new` underneath 'Parameter', type `color_scheme`, and click the 
+  `Create parameter` drop-down presented. Under 'Baseline', provide a value of `light` to 
+  `color_scheme`. Under 'Variant A', provide a value of `dark` to `color_scheme`. Finally, press 
+  the `Review` button.
```
Comment (ncooke3) [inline abtesting/README.md:30]: it looks like some additional whitespace got added at the beginning of these lines

File: abtesting/Design/SYMBOLS.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        - [body](#body-2)
+
+## ABTestingExampleApp
+```swift
+@main
+struct ABTestingExampleApp: App
+```
+main point of entry into app
+
+### appConfig
```
Comment (ncooke3) [inline abtesting/Design/SYMBOLS.md:36]: I left this idea on other PR as well:
Optional: consider making header an inline code block
>### `appConfig`
>```swift
>var appConfig: AppConfig
>```
>stores the AppConfig instance

**Review by ncooke3 — APPROVED** [review]

File: abtesting/README.md (hunk 1 above)
Comment (jrcrespoh) [reply abtesting/README.md:40]: That was intentional in that the previous character in the sentence is already at column 100, but this seeming whitespace doesn't show up in the Markdown rendering. Please let me know if you think I should tidy this up.

File: abtesting/README.md (hunk 2 above)
Comment (jrcrespoh) [reply abtesting/README.md:30]: Quoting above:
> That was intentional in that the previous character in the sentence is already at column 100, but this seeming whitespace doesn't show up in the Markdown rendering. Please let me know if you think I should tidy this up.

Comment (jrcrespoh) [reply abtesting/README.md:30]: Nevermind, it does appear that these are extra. Thanks for catching these!

Comment (jrcrespoh) [reply abtesting/README.md:30]: Fixed ✅

File: abtesting/Design/SYMBOLS.md (hunk 3 above)
Comment (jrcrespoh) [reply abtesting/Design/SYMBOLS.md:36]: Done ✅

**Review by paulb777 — APPROVED** [review]

**Pull request merged**